docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/install-and-configure-stripe-prerequisites.md

```diff
@@ -217,6 +217,102 @@ use Spryker\Zed\OauthClient\Communication\Plugin\Payment\AccessTokenPaymentAutho
 
 ```
 
+## Using Stripe in Marketplace context
```

Comment:
If'm running a marketplace, do I need to complete the prior steps in this document? Or just the ones in this section?